### ISSUE TITLE
Too much memory consumption

### ISSUE THREAD

**Issue by nntoan209** [Kunhao-Liu/StyleRF#4]
Hi, when I try to run the script train.sh, everything worked fine, but when I run the script train_feature.sh, my computer will kill the running process. After navigating, I found that the function _prepare_feature_data_ in _BlenderDataset_ class has some problem with the memory consumption, it uses so much RAM, more than 64GB RAM after running half of the number of iterations. Is this a bug or you just used more RAM in your experiments ?

**Comment by Kunhao-Liu** [comment]
Hi, the `prepare_feature_data()` function consumes a lot of memory because we store the feature map of every input view in memory. To reduce RAM consumption, you can use fewer input views for `train_feature.sh`.

**Comment by nntoan209** [comment]
Thanks for your reply. I have another question: What are the differences between the argument render_train, render_test and render_path? If I only want to render novel views for a scene with my custom style image, which option should I use? I came up with this question because when using test_style.sh for nerf_synthetic data, if I use the argument render_path, then there is a issuse that the BlenderDataset class doesn't have the render_path attribute

**Comment by Kunhao-Liu** [comment]
In the llff dataset, render_train renders the stylized views in the training set, render_test renders the test set, and render_path renders some continuous views that form a video. In the blender dataset, we only have render_train and render_test, the latter of which outputs a video.

**Comment by jly0810** [comment]
> Hi, the `prepare_feature_data()` function consumes a lot of memory because we store the feature map of every input view in memory. To reduce RAM consumption, you can use fewer input views for `train_feature.sh`.

您好 我也遇到了内存的问题 您能具体说一下解决方法吗 需要修改blender.py吗
您可以提供阶段二和三的blender预训练结果吗 不胜感激
